app/models/stage.rb

```diff
@@ -180,6 +180,11 @@ def deploy_group_names
     DeployGroup.enabled? ? deploy_groups.select(:name).sort_by(&:natural_order).map(&:name) : []
   end
 
+  def duplicable!
+    project.duplicable_stage && project.duplicable_stage.update!(duplicable: false)
```

Comment:
Label should then also say something like "Maximum of 1 per project"

On Fri, Aug 12, 2016 at 3:57 PM, Adam Panzer <notifications@github.com>
wrote:

> In app/models/stage.rb
> <https://github.com/zendesk/samson/pull/1198#discussion_r74671800>:
>
> > @@ -180,6 +180,11 @@ def deploy_group_names
> >      DeployGroup.enabled? ? deploy_groups.select(:name).sort_by(&:natural_order).map(&:name) : []
> >    end
> >
> > +  def duplicable!
> > +    project.duplicable_stage && project.duplicable_stage.update!(duplicable: false)
>
>
>   def duplicable!
>     old_duplicable_stage = project.duplicable_stage
>     old_duplicable_stage.update!(duplicable: false) if old_duplicable_stage
>     update!(duplicable: true)
>   end
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1198/files/122035a3f8635be3fd046f6620704db40db5d7c0#r74671800>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ2oo7wQoEyn0JqLsfnjH9HLXH2Fgks5qfPpNgaJpZM4Jhtog>
> .
>